### PR TITLE
Adds node chipid

### PR DESCRIPTION
**Pull request by stijnbernards**
Hi,

Just want to let you know that I think this is a really nice repository and I aim to add more code to it.
I'm currently using you code in my boilerplate setup:  https://github.com/stijnbernards/NodeMCU-Lua-Boilerplate
I hope this is okay with you.

Cheers

### CODE REVIEW

**Comment by fikin** [comment]
sure, i'm looking forward for more api support

File: lua/node.lua [hunk 1]
```diff
@@ -15,4 +15,18 @@ node.input = function(str)
   )
 end
 
+node.chipid = function()
```
Comment (fikin) [inline lua/node.lua:18]: shouldn't chipid be constant across one test case?
i'd suggest to mimic mdns implementation and move the variables (chipid and reason) and math.random to nodemcu-module re-assignable variables.

Comment (stijnbernards) [reply lua/node.lua:18]: Thanks! Seems like a better solution to me too I'll amend my PR.

**Comment by fikin** [comment]
closing until further actions are taken.

**Pull request closed without merging**